commands/get/templates_test.go

```diff
@@ -52,12 +59,14 @@ func TestTemplatesAction401Error(t *testing.T) {
 		templateType: "projects",
 		getter:       mock.CreateRequest(t, 401, "testing.feedhenry.me/box/api/templates/projects", response),
 	}
-	if err := tCmd.templatesAction(nil); err == nil {
+	if err := tCmd.templatesAction(cxt); err == nil {
 		t.Fatal("expected an error executing command")
 	}
 }
 
 func TestTemplatesAction500Error(t *testing.T) {
+	cxt := cli.NewContext(cli.NewApp(), flag.NewFlagSet("o", 0), nil)
```

Comment:
with this being needed in every test it may be a candidate for moving out to its own helper function